src/ocm/hooks/useClusterSupportedPlatforms.ts

```diff
@@ -41,8 +42,24 @@ export default function useClusterSupportedPlatforms(clusterId: string) {
   }, [error, addAlert, clusterId, alerts]);
 
   return {
-    isPlatformIntegrationSupported,
+    isPlatformIntegrationSupported: supportedPlatformIntegration !== undefined,
+    supportedPlatformIntegration: supportedPlatformIntegration
+      ? (supportedPlatformIntegration as SupportedPlatformIntegrationType)
+      : 'no-active-integrations',
     isLoading,
     error,
   };
 }
+
+function getUniqueValuesFromData(data: PlatformType[] | undefined) {
```

Comment:
I'd prefer something that's easier to understand in the business domain.
The function name etc doesn't reveal why it's doing something, just somehow that it's extracting some value etc.  Note that the function is not being called if `data` is not defined, which makes it shorter as well.

```
function getIntegrablePlatformIntegration(platform: PlatformType[]) {
  const platformsSet = new Set(platform);
  if (platformsSet.size === 1) {
    const exclusivelyAvailablePlatform = platformsSet.values().next()
      .value as PlatformIntegrationType;
    return platformsToIntegrate.includes(exclusivelyAvailablePlatform);
  }
  return undefined;
}
```

Reply:
Well seen. I'll change it